packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/generateDataURL.ts

```diff
@@ -0,0 +1,49 @@
+import getGeneratedImageSize from './getGeneratedImageSize';
+import ImageEditInfo from '../types/ImageEditInfo';
+
+/**
+ * @internal
+ * Generate new dataURL from an image and edit info
+ * @param image The image to generate data URL from. It is supposed to have original src loaded
+ * @param editInfo Edit info of the image
+ * @returns A BASE64 encoded string with image prefix that represents the content of the generated image.
+ * If there are rotate/crop/resize info in the edit info, the generated image will also reflect the result.
+ * It is possible to throw exception since the original image may not be able to read its content from
```

Comment:
As this function may throw an exception, do you think it's good idea to include the [@throws](https://jsdoc.app/tags-throws.html) tag in the function documentation?